From 48a15f722be7c26010332536df1ba5bd029c6843 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timi=20Rautam=C3=A4ki?= <timi.rautamaki@gmail.com>
Date: Sun, 7 Nov 2021 11:17:34 +0000
Subject: [PATCH 03/22] Launcher3: Fix all apps view not dismissing

The logic was reversed. All apps view should dissmiss on back key when
the searchbox is focused but empty and clear when not empty.

Change-Id: Id11c991dae6c159b986966f278f0c792993b6abd
---
 .../launcher3/allapps/search/AllAppsSearchBarController.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/launcher3/allapps/search/AllAppsSearchBarController.java b/src/com/android/launcher3/allapps/search/AllAppsSearchBarController.java
index 4427a49dab..f94658c404 100644
--- a/src/com/android/launcher3/allapps/search/AllAppsSearchBarController.java
+++ b/src/com/android/launcher3/allapps/search/AllAppsSearchBarController.java
@@ -132,7 +132,7 @@ public class AllAppsSearchBarController
     public boolean onBackKey() {
         // Only hide the search field if there is no query
         String query = Utilities.trim(mInput.getEditableText().toString());
-        if (query.isEmpty()) {
+        if (!query.isEmpty()) {
             reset();
             return true;
         }
-- 
2.34.1

